app/components/app_patient_table_component.rb

```diff
@@ -15,6 +15,19 @@ def initialize(patients, current_user:, count:)
 
   def can_link_to?(record) = allowed_ids.include?(record.id)
 
+  def unlinkable_patient_hint_text(patient)
```

Comment:
I'm a little worried about potential N+1 performance issues with this, I think both `patient.teams` and `patient.sessions` has the potential to make requests to the database. If we have an uploaded file with a lot of patients that are not linkable this might be a problem.

I'm not sure it's something we need to worry about right now, but worth noting.